lib/instadeploy.py

```diff
@@ -184,16 +186,26 @@ def build():
     if response.status_code != requests.codes.ok:
         raise MxBuildFailure("MxBuild failure", response.status_code, response.json())
 
+    result = response.json()
+    if result['status'] == 'Success':
+        try:
+            sync_project_files()
+            logger.info("Syncing project files ...")
```

Comment:
From testability (which we do not have btw), the current change 
makes it easy to write a test case to be able to catch the issues from
`sync_project_files()`.

From readability, you are right. 

I leaned towards the first one. I can fix this in the next PR?